doc/nrf/releases_and_maturity/abi_compatibility.rst

```diff
@@ -99,7 +100,7 @@ Fixed
 * An issue where permission checks for pointer members in the SSF PSA crypto service requests were incorrect.
 * An issue with invoking crypto service from multiple threads or clients.
 
-System Controller Firmware (SCFW) v4.0.2
+System Controller Firmware (SCFW) v4.0.3
```

Comment:
* LRC uses direct gdpwr request.
* Workaround - USB D+ pull-up glitch

Reply:
@hakonfam Is there a previous entry for the issue fixed by this workaround in the known issues page?

Reply:
No, it was discovered after that.